table_loading.py

```diff
@@ -344,13 +344,15 @@ def mag_to_flux(tbl, magcols, emagcols, zpts, filternames):
     for colname, errcolname, zpn in zip(magcols, emagcols, filternames):
         print(colname, zpn)
         zp = u.Quantity(zpts[zpn], u.Jy)
-        data = tbl[colname].value
-        if hasattr(tbl[colname], 'mask'):
-            tbl[zpn+"_flux"] = flx = np.ma.masked_where(tbl[colname].mask, (zp * 10**(data/-2.5)).to(u.mJy))
-        else:
-            tbl[zpn+"_flux"] = flx = (zp * 10**(data/-2.5)).to(u.mJy)
-        err = tbl[errcolname] / (1.09*u.mag) * flx
-        tbl[zpn+"_eflux"] = err
+        if colname in tbl.keys():
```

Comment:
I think this simplification should work:
```suggestion
        if colname in tbl:
```